src/winter_onboarding_2021/fleet_management_service/models/organisation.clj

```diff
@@ -0,0 +1,11 @@
+(ns winter-onboarding-2021.fleet-management-service.models.organisation
+  (:require [winter-onboarding-2021.fleet-management-service.db.organisation :as org-db]
+            [winter-onboarding-2021.fleet-management-service.models.user :as user-models]))
+
+(defn create [org]
+  (org-db/create org))
+
+(defn create-and-associate [org admin]
+  (let [db-org (create (assoc org
+                              :created-by (:users/id admin)))]
+    (user-models/add-user-to-org db-org admin)))
```

Comment:
I am not sure, but shouldn't this be a single transaction? (Updating users table and creating new record in organizations)

Reply:
Ideally, it should be. Will wrap them in a transaction